src/axom/primal/operators/winding_number.hpp

```diff
@@ -109,16 +110,19 @@ int winding_number(const Point<T, 2>& q,
 template <typename T>
 int winding_number(const Point<T, 2>& R,
                    const Polygon<T, 2>& P,
+                   bool& isOnEdge,
                    bool includeBoundary = false,
                    double EPS = 1e-8)
```

Comment:
Suggestion: To avoid accidental overload ambiguities, it might be a good idea for this version to be more explicit and not have any default parameters.
E.g.
```suggestion
int winding_number(const Point<T, 2>& R,
                   const Polygon<T, 2>& P,
                   bool& isOnEdge,
                   bool includeBoundary,
                   double EPS)
```